ecommerce_api_client/client.py

```diff
@@ -22,7 +22,7 @@ def __init__(self, url, signing_key=None, username=None, full_name=None, email=N
         if oauth_access_token:
             auth = BearerAuth(oauth_access_token)
         elif signing_key and username:
-            auth = JwtAuth(username, full_name, email, signing_key, tracking_context)
+            auth = JwtAuth(username, full_name, email, signing_key, tracking_context, issuer, expires_in)
```

Comment:
This works, but since `tracking_context`, `issuer`, and `expires_in` are keyword arguments, it would be clearer if they were provided as such.
```python
auth = JwtAuth(
    username,
    full_name,
    email,
    signing_key,
    issuer=issuer,
    expires_in=expires_in,
    tracking_context=tracking_context
)
```